terraform/modules/digitalocean/database/README.md

```diff
@@ -0,0 +1,56 @@
+## Requirements
```

Comment:
Can you say what this module _is_? Does it provision both MySQL and MongoDB, for example?

Reply:
It is used for creating database clusters for DigitalOcean. In case of DO, the only differentiator between MySQL/MongoDB/Postgres is the "engine" and "engine version".